specifyweb/frontend/js_src/lib/components/Statistics/Categories.tsx

```diff
@@ -292,6 +297,28 @@ export function Categories({
           {statsText.addACategory()}
         </Button.Secondary>
       )}
+      {typeof deletingCategory === 'number' && (
```

Comment:
this component is already large - it would be easy to move the button and this dialog into a separate component

Reply:
Moved to a new file!